metals/src/main/scala/scala/meta/internal/metals/codelenses/RunTestCodeLens.scala

```diff
@@ -73,17 +73,26 @@ final class RunTestCodeLens(
       // most of the bsp servers such as bloop and sbt might not support it.
     } yield {
       val classes = buildTargetClasses.classesOf(buildTargetId)
+      // sbt doesn't declare debugging provider
```

Comment:
I'm assuming this is because the plugin is what is actually providing this functionality right? If so, is there no way for the plugin to set this value? It's sort of a bummer that we have to have this extra check for certain tools. I know this isn't new code, but seeing this just made me think of it again.

Reply:
Yeah, that's something that I was also wondering @adpi2 any idea if we can change capabilities in the plugin?

Reply:
Currently I don't think it is. But in the future I should open the BSP implementation in sbt so that plugins can override it.